docs/chat-configuration/ui-customization/incoming-message/incoming-options.md

```diff
@@ -85,24 +85,24 @@ Incoming bot response can have several options to the user to choose from. Those
     ```
 
 ## Channels
-- Channels are a sub type of QuickOptions. Channels are used for user escalation actions.   
-- Channels may appear as response options or on article page.  
-- Channels can be created on the [Bold360ai console](https://support.bold360.com/ai).
+- Channels are a sub type of QuickOptions. Channels provide  the user with escalation paths.   
+- Channels may appear as a response options or as part of an article page.  
+- Channels are defined in the [Bold360ai console](https://support.bold360.com/ai).
 
-### <U>Customizing channels icons</U>
-- #### By setting the icons via the Bold360ai console:
+### <U>Customizing the channel's icon</U>
+- #### Via the Bold360ai console:
     ![]({{'/assets/images/console-channeling-icons.png' | relative_url}})
     {: .image-40}
 
-- #### By overriding SDK default icons. 
+- #### Via the SDK, by overriding the SDK's default icons. Two options are available as see bellow: 
```

Comment:
instead of "####" use the following below the line:
{: .strong-sub-title}
i think there should be a line break after "... default icons."

Reply:
OK